authors/apps/analytics/models.py

```diff
@@ -11,17 +11,23 @@ class ReadsReport(TimeStampModel):
     Display no of users who clicked on an article.
     """
     article = models.ForeignKey(Articles, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
     # Shows if user has read the entire article.
     full_read = models.BooleanField(default=False)
+    num_views = models.IntegerField(null=False)
```

Comment:
Great stuff! The views account for both the anonymous user view as well as authenticated users. I would recommend changing the name of `num_views` to `number_of_views`